torch/_refs/__init__.py

```diff
@@ -704,6 +713,24 @@ def sub(
     aten_op=None,  # CompositeImplicitAutograd
 )
 
+
+def _xlogy(a: Union[Tensor, NumberType], b: Union[Tensor, NumberType]):
+    if isinstance(a, Tensor) and isinstance(b, Number):
```

Comment:
Add a comment here, please -- is this because torch.xlogy doesn't support scalar inputs?

Reply:
`torch.xlogy` supports scalar inputs but `torch.log` does not.

Reply:
Of course, that makes sense -- let's add a TODO comment here that we should make refs.log (and other elementwise unary ops) support scalar inputs?